frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -364,6 +365,12 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             icon: <IconGraph />,
                             logic: isUsingSidebar ? insightsSidebarLogic : undefined,
                             to: isUsingSidebar ? undefined : urls.savedInsights(),
+                            sideAction: {
+                                icon: <IconPlusMini />, // The regular plus is too big
```

Comment:
@corywatilo `IconPlus` in @posthog/icons is really big! If that's Central's rendition of a plus, then I think it'd be practical to make it just a bit smaller. WDYT?